<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLVideoCameraIntrinsicsWidget</class>
 <widget class="QWidget" name="qMRMLVideoCameraIntrinsicsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>530</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ctkCollapsibleButton" name="collapsibleButton_Camera">
     <property name="text">
      <string>Camera</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="qMRMLNodeComboBox" name="comboBox_CameraSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLVideoCameraNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkCollapsibleButton" name="collapsibleButton_Details">
        <property name="text">
         <string>Details</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_IntrinMatrix">
           <property name="text">
            <string>Intrinsics Matrix:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="container_Intrinsics" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="1" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1" rowspan="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QTableWidget" name="tableWidget_CameraMatrix">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="gridStyle">
               <enum>Qt::DotLine</enum>
              </property>
              <property name="rowCount">
               <number>3</number>
              </property>
              <property name="columnCount">
               <number>3</number>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <row/>
              <row/>
              <row/>
              <column/>
              <column/>
              <column/>
              <item row="0" column="0">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="0" column="2">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="1" column="0">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="1" column="1">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="1" column="2">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="2" column="0">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="2" column="1">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="2" column="2">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_DistCoeffs">
           <property name="text">
            <string>Distortion Coefficients:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QWidget" name="container_DistCoeffs" native="true">
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="1" column="0">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QTableWidget" name="tableWidget_DistCoeffs">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="gridStyle">
               <enum>Qt::DotLine</enum>
              </property>
              <property name="rowCount">
               <number>1</number>
              </property>
              <property name="columnCount">
               <number>5</number>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <row/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <item row="0" column="0">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="0" column="2">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="0" column="3">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="0" column="4">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_Tracker">
           <property name="text">
            <string>MarkerToImageSensor:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QWidget" name="container_MarkerToImage" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="1" column="0">
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1" rowspan="2">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QTableWidget" name="tableWidget_MarkerToImageSensor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="gridStyle">
               <enum>Qt::DotLine</enum>
              </property>
              <property name="rowCount">
               <number>4</number>
              </property>
              <property name="columnCount">
               <number>4</number>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <row/>
              <row/>
              <row/>
              <row/>
              <column/>
              <column/>
              <column/>
              <column/>
              <item row="0" column="0">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="0" column="2">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="0" column="3">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="1" column="0">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="1" column="1">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="1" column="2">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="1" column="3">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="2" column="0">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="2" column="1">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="2" column="2">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="2" column="3">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="3" column="0">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="3" column="1">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="3" column="2">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="3" column="3">
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_CameraPlaneOffset">
           <property name="text">
            <string>Camera Plane Offset (mm):</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QWidget" name="container_CameraPlaneOffset" native="true">
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="1" column="0">
             <spacer name="verticalSpacer_7">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1" rowspan="2">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QTableWidget" name="tableWidget_CameraPlaneOffset">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="rowCount">
               <number>1</number>
              </property>
              <property name="columnCount">
               <number>3</number>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <row/>
              <column/>
              <column/>
              <column/>
              <item row="0" column="0">
               <property name="text">
                <string>0.0</string>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string>0.0</string>
               </property>
              </item>
              <item row="0" column="2">
               <property name="text">
                <string>0.0</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
